integration/spark/app/src/test/java/io/openlineage/spark/agent/SparkGenericIntegrationTest.java

```diff
@@ -8,6 +8,7 @@
 import static io.openlineage.spark.agent.MockServerUtils.getEventsEmitted;
 import static io.openlineage.spark.agent.MockServerUtils.verifyEvents;
 import static io.openlineage.spark.agent.SparkTestUtils.SPARK_VERSION;
+import static org.apache.spark.sql.functions.*;
```

Comment:
Let's keep explicit imports, please.

This helps if you're using IntelliJ: https://www.baeldung.com/intellij-disable-wildcard-import